From 2b57f91cbd203d8d9557513376b7848fb408385a Mon Sep 17 00:00:00 2001
From: Chris Storah <chris.storah@gmail.com>
Date: Fri, 8 Mar 2024 17:46:56 +1100
Subject: [PATCH] abc

---
 drivers/media/i2c/Kconfig | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)

diff --git a/drivers/media/i2c/Kconfig b/drivers/media/i2c/Kconfig
index 74ff833ff48c..d985fd20cde7 100644
--- a/drivers/media/i2c/Kconfig
+++ b/drivers/media/i2c/Kconfig
@@ -200,6 +200,29 @@ config VIDEO_IMX415
          To compile this driver as a module, choose M here: the
          module will be called imx415.
 
+config VIDEO_IMX708
+       tristate "Sony IMX708 sensor support"
+       depends on I2C && VIDEO_DEV
+       select MEDIA_CONTROLLER
+       select VIDEO_V4L2_SUBDEV_API
+       select V4L2_FWNODE
+       help
+         This is a Video4Linux2 sensor driver for the Sony
+         IMX708 camera.
+
+         To compile this driver as a module, choose M here: the
+         module will be called imx708.
+
+config VIDEO_AR0144
+       tristate "ON Semiconductor AR0144 sensor support"
+       depends on I2C && VIDEO_DEV
+       select MEDIA_CONTROLLER
+       select VIDEO_V4L2_SUBDEV_API
+       select V4L2_FWNODE
+       help
+         This is a V4L2 sensor-level driver for the ON Semiconductor AR0144
+         CMOS image sensor.
+
 config VIDEO_MAX9271_LIB
        tristate
 
-- 
2.39.2

