diff --git a/arch/arm64/boot/dts/freescale/imx8mn-ddr4-evk-ap1302.dts b/arch/arm64/boot/dts/freescale/imx8mn-ddr4-evk-ap1302.dts
new file mode 100644
index 000000000..4aa91dad2
--- /dev/null
+++ b/arch/arm64/boot/dts/freescale/imx8mn-ddr4-evk-ap1302.dts
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include "imx8mn-ddr4-evk.dts"
+
+&i2c3 {
+        /delete-node/ov5640_mipi@3c;
+
+        ap1302_isp@3C {
+                compatible = "onsemi,ap1302";
+                #address-cells = <1>;
+                #size-cells = <0>;
+                reg = <0x3C>;
+                csi_id = <0x00>;
+                mipi_csi;
+                status = "okay";
+                pinctrl-names = "default";
+                pinctrl-0 = <&pinctrl_camera>;
+                powerdown-gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
+                reset-gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+
+                port {
+                        camera_ep_0: endpoint {
+                                data-lanes = <1 2 3 4>;
+                                clock-lanes = <0>;
+                                /* link-frequencies = /bits/ 64 <750000000>; */
+                                /* max-lane-frequency = /bits/ 64 <750000000>; */
+                                /* max-pixel-frequency = /bits/ 64 <500000000>; */
+                                /* max-data-rate = /bits/ 64 <0>; */
+                                remote-endpoint = <&mipi1_sensor_ep>;
+                        };
+                };
+        };
+};
+
+
+&mipi_csi_1 {
+        #address-cells = <1>;
+        #size-cells = <0>;
+        clock-frequency = <240000000>;
+        status = "okay";
+
+        port@0 {
+                mipi1_sensor_ep: endpoint {
+                        remote-endpoint = <&camera_ep_0>;
+                        data-lanes = <4>;
+                        csis-hs-settle = <11>;
+                        csis-clk-settle = <0>;
+			csis-wclk;
+                };
+        };
+};
+
+
+&isi_0 {
+	status = "okay";
+};
+
+&cameradev {
+    status = "okay";
+};
