From d0954ac08c820387fd496006a96c57686cb8a6a6 Mon Sep 17 00:00:00 2001
From: Chris Storah <chris.storah@gmail.com>
Date: Fri, 5 Jan 2024 03:24:59 +0000
Subject: [PATCH] add ap1302 dtb into makefile

---
 arch/arm64/boot/dts/freescale/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 4b49044bff20..0b3cbf705f4c 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -104,6 +104,7 @@ dtb-$(CONFIG_ARCH_MXC) += imx8mn-ddr3l-evk.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mn-ddr4-evk.dtb imx8mn-ddr4-evk-rm67191.dtb imx8mn-ddr4-evk-rm67199.dtb \
 			  imx8mn-ddr4-evk-rm67191-cmd-ram.dtb imx8mn-ddr4-evk-rm67199-cmd-ram.dtb \
 			  imx8mn-ddr4-evk-usd-wifi.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mn-ddr4-evk-ap1302.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mn-ddr4-evk-rpmsg.dtb imx8mn-ddr3l-evk-rpmsg.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mn-ddr4-evk-root.dtb imx8mn-ddr4-evk-inmate.dtb imx8mn-evk-root.dtb imx8mn-evk-inmate.dtb \
 			  imx8mn-evk-lk.dtb imx8mn-ddr4-evk-lk.dtb
-- 
2.39.2

