From 609796bf49fa0ac825310c6d5f5e618c5c0833da Mon Sep 17 00:00:00 2001
From: Chris Storah <chris.storah@gmail.com>
Date: Wed, 24 Jan 2024 16:28:20 +1100
Subject: [PATCH] added debug to ap1302

---
 drivers/media/i2c/ap1302.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/ap1302.c b/drivers/media/i2c/ap1302.c
index b4fa0ba2b44d..d1ace9c80d46 100644
--- a/drivers/media/i2c/ap1302.c
+++ b/drivers/media/i2c/ap1302.c
@@ -7,7 +7,7 @@
  * Copyright (C) 2021, Laurent Pinchart <laurent.pinchart@ideasonboard.com>
  *
  */
-
+#define DEBUG 1
 #include <linux/clk.h>
 #include <linux/debugfs.h>
 #include <linux/delay.h>
-- 
2.39.2

